openslides/motions/static/js/motions/site.js

```diff
@@ -1119,6 +1127,26 @@ angular.module('OpenSlidesApp.motions.site', [
         $scope.reset_state = function () {
             $http.put('/rest/motions/motion/' + motion.id + '/set_state/', {});
         };
+        // save additional state field
+        $scope.saveAdditionalStateField = function (stateExtension) {
+            if (stateExtension) {
+                motion["comment " + $scope.commentFieldForState] = stateExtension;
+                motion.title = motion.getTitle(-1);
+                motion.text = motion.getText(-1);
+                motion.reason = motion.getReason(-1);
+                Motion.save(motion);
```

Comment:
yes, this function should save only the additional state (or recommendationI) comment field. But the server requires that title, text and reason is set to save other values (like comments). Do you have any hint to improve it?